### PR TITLE
Misc Updates to PublishingModal

### PR DESCRIPTION
**Pull request by ThomasThelen**
There are a couple misc changes in here, mostly css changes

# Change 1

Misc styling things like the font color and alignment changes.

# Change 2

I added the `science_metadata.xml` file to the Environment section. Should complete #171 

# Change 3

The dialog now listens on the event stream for any publishing notifications.

Relevant commits are

3adab8dbd72defd40e7a8bcb449ef9ae6371b62a
afd66674701d108142d9553054031557840e25bf
0e8cf4025cc4bdffeeffd7e2d00b62bd713292bb
3aed0bc6fd2333a6852eb4420b7b12b7313a30fe
b3a375d168cc8f0bb3ea43e8a95a37c1d775335b

### CODE REVIEW

**Comment by ThomasThelen** [comment]
screen shot 2018-08-06 at 9 11 03 am
screen shot 2018-08-06 at 9 10 59 am


How it looks after this PR

File: app/components/ui/files/publish-modal/component.js [hunk 1]
```diff
@@ -17,18 +19,44 @@ export default Ember.Component.extend({
...
+        // Create the tooltips after the template has been rendered
+        this.create_tooltips();
+    },
 
     setPublishBtnState(state) {
         this.set('enablePublish', state);
     },
 
+    setTaleName() {
+        let url = config.apiUrl + '/tale/'+ this.get('modalContext')
```
Comment (amoeba) [inline app/components/ui/files/publish-modal/component.js]: Feel free to not trouble yourself over this as my Ember experience is very much in progress. I remember asking others on Slack for the most Emberish way to do this so feel free to disregard my ramblings if this is just the way others are doing this type of stuff.

Does Ember give us a handy-dandy way to do this without (1) the extra call to `/tale` and without having to construct the URL from scratch? Re: (1), I wonder if the underlying view can share the already-fetched Tale instance with the modal to save the extra call. Re: (2) I'm used to applications using reflection of some sort to dynamically determine routes so the code is resilient to changes in URL structure and, in this case, delegating the responsibility of determining the URL and doing the querying to other parts of the application. It'd result in code like:

```js
setTaleName() {
  var tale = new Tale(this.get('modalContext')) # Handles the request
  self.set(tale.get('title'))
}
```

**Comment by amoeba** [comment]
This is all looking great. Is the icon that was chosen for the  Usage Rights section of the modal intentional?

Since #222 and #223 contain so many related/shared commits across similar paths, what do you think about either combining these into a single PR or something? It's totally fine if, in the future, if you want to keep pushing new commits to an existing PR if the commits are tightly related.

**Comment by ThomasThelen** [comment]
That icon from semanticui is broken (I tried it in a new ember application) so I'm going to switch them over to font awesome (we're moving away from semanticui)

**Comment by ThomasThelen** [comment]
Moving to a new PR

**Pull request closed without merging**